src/interpolate/interpolate_kernel.cu

```diff
@@ -157,9 +157,13 @@ __global__ void interpolate_backward_kernel(
         bary_img_grad.data + bary_img_grad_sN * n + bary_img_grad_sH * h + bary_img_grad_sW * w;
 
     bool thread_is_used = tr_index != -1;
-
+#ifndef __HIP_PLATFORM_AMD__
+    unsigned m = 0xFFFFFFFFU;
+#else
+    uint64_t m = 0xFFFFFFFFFFFFFFFFull;
```

Comment:
This change is done because MI300X is based on the gfx9 arch, which uses 64 threads per wave (warp in NVIDIA jargon). This means the mask has to be 64 bits instead of 32.